library/src/animationManager.cxx

```diff
@@ -267,8 +267,11 @@ bool animationManager::LoadAtTime(double timeValue)
     // Set progress bar
     vtkProgressBarRepresentation* progressRep =
       vtkProgressBarRepresentation::SafeDownCast(this->ProgressWidget->GetRepresentation());
-    progressRep->SetProgressRate(
-      (this->CurrentTime - this->TimeRange[0]) / (this->TimeRange[1] - this->TimeRange[0]));
+    if (this->Options->getAsBool("ui.animation-progress"))
+      progressRep->SetProgressRate(
+        (this->CurrentTime - this->TimeRange[0]) / (this->TimeRange[1] - this->TimeRange[0]));
+    else
+      progressRep->SetProgressRate(0);
```

Comment:
You may want to use `progressRep->SetVisibility(false);` instead

Reply:
You mean like this, right?
```
progressRep->SetProgressRate(
        (this->CurrentTime - this->TimeRange[0]) / (this->TimeRange[1] - this->TimeRange[0]));
else
      progressRep->SetVisibility(false);
```
I'll change that.

Reply:
Is there a way to reload configuration without relaunching?

Reply:
I know have realized that it is much better to simple leaved progress widget off during `animationManager::Initialize`.

Reply:
hum, does that let you turn it on and off during execution ?

Im not sure that would be the case.

Reply:
> Is there a way to reload configuration without relaunching?

If you mean is there a way to change the config without running from the start, yes there is. When you change a file, its own config block is read, which could be different from the other file and enable/disable the animation bar.

Also libf3d user may do other things.

This is why libf3d option have a "Trigger": https://f3d.app/doc/libf3d/OPTIONS.html

This is how the option is being "updated", some option are updated on load aka, when loading a file. some options are updated on "render", aka, when rendering again. No options should only be updated on "start".

Reply:
If I remember well, "Initialize" is called on loadFile, which makes this option updated on "load", which is fine :)